cmd/devtool/README.md

```diff
@@ -10,12 +10,12 @@ An on-chain workflow testing tool that supports the following:
 ## Step 1: Set up a private ETH network with Livepeer protocol deployed
 
 ```
-docker pull livepeer/geth-with-livepeer-protocol:streamflow
-docker run -p 8545:8545 -p 8546:8546 --name geth-with-livepeer-protocol livepeer/geth-with-livepeer-protocol:streamflow
+docker pull livepeer/geth-with-livepeer-protocol:confluence
+docker run -p 8545:8545 -p 8546:8546 --name geth-with-livepeer-protocol livepeer/geth-with-livepeer-protocol:confluence
 
 # Mac M1 ONLY
-# docker pull darkdragon/geth-with-livepeer-protocol:streamflow
-# docker run -p 8545:8545 -p 8546:8546 --name geth-with-livepeer-protocol darkdragon/geth-with-livepeer-protocol:streamflow
+# docker pull darkdragon/geth-with-livepeer-protocol:confluence
+# docker run -p 8545:8545 -p 8546:8546 --name geth-with-livepeer-protocol darkdragon/geth-with-livepeer-protocol:confluence
```

Comment:
```suggestion
# docker pull darkdragon/geth-with-livepeer-protocol:streamflow
# docker run -p 8545:8545 -p 8546:8546 --name geth-with-livepeer-protocol darkdragon/geth-with-livepeer-protocol:streamflow
```

It's not correct, because `darkdragon` is not the Livepeer-owned Docker registry. It'd keep for now `streamflow` for M1 (arm64) and we'll soon fix it in https://github.com/livepeer/go-livepeer/issues/2316